decidim-comments/app/frontend/comments/comment.component.tsx

```diff
@@ -209,7 +268,7 @@ class Comment extends React.Component<CommentProps, CommentState> {
   private _renderVoteButtons() {
     const { session, comment, votable, rootCommentable, orderBy } = this.props;
 
-    if (votable) {
+    if (session && votable) {
```

Comment:
Another "variable could expectedly be nil" error (don't remember the exact copy but it can be reproduced by removing the check :))